packages/twenty-front/src/modules/object-record/record-field/hooks/__tests__/useIsFieldEmpty.test.tsx

```diff
@@ -49,6 +49,7 @@ describe('useIsFieldEmpty', () => {
       });
     });
 
+    // Todo: fix this test
     expect(result.current.isFieldEditModeValueEmpty).toBe(false);
```

Comment:
@lucasbordeau we need to fix this: the new context selector is not there in tests